src/migrations/hotbar-store/5.0.0-alpha.0.ts

```diff
@@ -20,34 +20,24 @@
  */
 
 // Cleans up a store that had the state related data stored
-import type { Hotbar } from "../../common/hotbar-store";
-import { ClusterStore } from "../../common/cluster-store";
+import { Hotbar, HotbarStore } from "../../common/hotbar-store";
 import * as uuid from "uuid";
 import type { MigrationDeclaration } from "../helpers";
+import { catalogEntity } from "../../main/catalog-sources/general";
 
 export default {
   version: "5.0.0-alpha.0",
```

Comment:
I believe that this is the best place to add this hotbar entity even though this is slightly anachronistic.

That is because for all non-beta users this is guaranteed to run and simplifies the logic since before `alpha.0` we know that there was not a hotbar in existence.